src/CaptureHigherEd/LaravelJira/Models/Field.php

```diff
@@ -90,6 +92,28 @@ public function getClauseNames()
         return $this->clauseNames;
     }
 
+    public function getOptions(string $project_key, string $issue_type_name)
+    {
+        $jira = app(Jira::class);
```

Comment:
This is a bit odd to do. It feels like it doesnt belong inside of the Field class since it relates to the issues API instead of fields. If you add it over there, you can have it return the entire array then use a filter to squash it down to just the 1 fields options.